integrations/axum/src/lib.rs

```diff
@@ -1214,3 +1141,13 @@ impl LeptosRoutes for axum::Router {
         router
     }
 }
+
+fn get_leptos_pool() -> LocalPoolHandle {
+    static LOCAL_POOL: OnceLock<LocalPoolHandle> = OnceLock::new();
+    LOCAL_POOL
+        .get_or_init(|| {
+            let pool = tokio_util::task::LocalPoolHandle::new(1);
```

Comment:
I will mention, at the moment there's only a single thread in the pool. We might want to consider making this configurable, or matching the existing tokio runtime settings.

Reply:
Yes! How about a call to https://doc.rust-lang.org/std/thread/fn.available_parallelism.html so we create one for however many threads are available